config/locales/client.en.yml

```
@@ -4118,6 +4118,7 @@ en:
       granted:
         one: "%{count} granted"
         other: "%{count} granted"
+      granted_text_only: "granted"
```

I suggest moving a simplified version of the HTML into the translation in order to make the whole string translatable. It's not optimal, but better than having an untranslatable string. How about this?

```suggestion
      granted_with_count:
        one: '<a href="%{badgeUrl}" title="%{count} granted" class="grant-count">%{count}x</a> <span class="grant-text">granted</span>'
        other: '<a href="%{badgeUrl}" title="%{count} granted" class="grant-count">%{count}x</a> <span class="grant-text">granted</span>'
```

---

I don't think works because this string is used elsewhere without the x

---

`granted_text_only` which you added in this PR isn't not used anywhere else as far as I can see. `granted_with_count` would simply replace `granted_text_only`.